data_transfer/dags/btf.py

```diff
@@ -17,24 +25,59 @@ def dag(study_site: StudySite) -> None:
             ->task_prepare_data
         ->batch_upload_data
 
+    Parameters
+    ----------
+    study_site : StudySite
+    days : int, optional
+        The amount of days in the past to query data for. This defaults to 50 days
+        to ensure prior data which has been delayed in upload is picked up
+    delta : int, optional
+        Initial reference (0 == today) to query backwards from. This allows
+        traversing back in time for historical data
+
+
     NOTE/TODO: this method simulates the pipeline.
     """
+    byteflies = Byteflies(study_site)
+
+    data_period = get_period_by_days(int(delta), int(days))
```

Comment:
How come `get_period_by_days` accepts ints and returns strings, both of which are casted to and from?

I'd change `get_period_by_days` to accept and returns ints, then modify `batch_metadata` so the cast occurs once.

Reply:
I'm casting here because due to our CLI use of this library, `delta` and `days` might be string